index.html

```diff
@@ -57,178 +67,188 @@ <h3>LET’S CONNECT</h3>
-          <li class="margin-spacer">JavaScript</li>
-        </ul>
-        <button class="margin-spacer">See Project</button>
       </div>
       <div class="grid-item">
         <div class="image-wrapper">
           <img src="/images/tonic.png" alt="work-image" />
         </div>
-
-        <h2>Tonic</h2>
-        <div class="work-details">
-          <p class="first-detail">CANOPY</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">Back End Dev</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">2015</p>
+        <div class="card-divider">
+          <h2>Tonic</h2>
+          <div class="work-details">
+            <p class="first-detail">CANOPY</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">Back End Dev</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer" type="button">See Project</button>
         </div>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <ul>
-          <li>HTML</li>
-          <li class="margin-spacer">CSS</li>
-          <li class="margin-spacer">JavaScript</li>
-        </ul>
-        <button class="margin-spacer" type="button">See Project</button>
       </div>
       <div class="grid-item">
         <div class="image-wrapper">
           <img src="/images/tonic.png" alt="work-image" />
         </div>
-
-        <h2>Tonic</h2>
-        <div class="work-details">
-          <p class="first-detail">CANOPY</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">Back End Dev</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">2015</p>
+        <div class="card-divider">
+          <h2>Tonic</h2>
+          <div class="work-details">
+            <p class="first-detail">CANOPY</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">Back End Dev</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer">see Project</button>
         </div>
+      </div>
```

Comment:
- Kindly make sure your card design stick as much as possible to the Figma design,  make sure that you reduce the space between the left and right sides of your card and add the height of your card and reduce the width to make it stick to the Figma. please check the comparison below :+1: 


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/236339197-3e4edb89-2ef0-459b-bb77-5283dc113020.png)

## You design 


![image](https://user-images.githubusercontent.com/95297251/236336528-9d79bf61-dba2-4c53-8d5b-c803d7774d51.png)